rllib/algorithms/dreamerv3/README.md

```diff
@@ -1,27 +1,94 @@
+predict a correct value function (based on the world model-predicted rewards), whereas
+the actor tries to come up with good actions to take for maximizing accumulated rewards
+over time.
+In other words, the actual RL components of the model (actor and critic) are never
+trained on real environment data, but on dreamed trajectories only.
+
+## Note on Hyperparameter Tuning for DreamerV3
+DreamerV3 is an extremely versatile and stable algorithm that not only works well on
+different action- and observation spaces, including discrete and continuous actions, as well
+as image and vector observations, but also almost needs no hyperparameter tuning, except for
+a simple "model size" setting (from "XS" to "XL") and a value for the training ratio, which
+specifies how many steps to replay from the buffer for a training update vs how many
+steps to take in the actual environment.
+
+
+## Note on multi-GPU Training with DreamerV3
+We found that when using multiple GPUs for DreamerV3 training, the following simple
+adjustments should be made to the standard config.
+- Multiply the original batch size (B=16) by the number of GPUs you are using.
+- Multiply the number of environments you sample from in parallel by the number of GPUs you are using.
+- Use a learning rate schedule for all learning rates (world model, actor, critic) with "priming".
+  - In particular, the first 10k timesteps should use low rates of `0.4` times of the published rates
+    (i.e. world model: `4e-5`, critic and actor: `1.2e-5`). 
+  - Over the course of the next 10k timesteps, linearly increase all rates to
+    n times their published values, where `n=max(4, [num GPUs])`.
+
+
+## Example Configs and Command Lines
+Use the config examples and templates in
+[the tuned_examples folder here](https://github.com/ray-project/ray/tree/master/rllib/tuned_examples/dreamerv3)
+in combination with the following scripts and command lines in order to run RLlib's DreamerV3 algorithm in your experiments:
+
+### Atari100k
+```shell
+$ cd ray/rllib/tests
+$ python run_regression_tests.py --dir ../tuned_examples/dreamerv3/atari_100k.py --env ALE/Pong-v5 
+```
+
+### DeepMind Control Suite (vision)
+```shell
+$ cd ray/rllib/tests
+$ python run_regression_tests.py --dir ../tuned_examples/dreamerv3/dm_control_suite_vision.py --env DMC/cartpole/swingup 
+```
+Other `--env` options for the DM Control Suite would be `--env DMC/hopper/hop`, `--env DMC/walker/walk`, etc..
+Note that you can also switch on WandB logging with the above script via the options
+`--wandb-key=[your WandB API key] --wandb-project=[some project name] --wandb-run-name=[some run name]`
+
+
+## Results
+Our results on the Atari 100k and (visual) DeepMind Control Suite benchmarks match those
+reported in the paper.
+
+### Pong-v5 (100k) 1GPU vs 2GPUs vs 4GPUs
+<img src="https://github.com/ray-project/ray/tree/master/doc/source/rllib/images/dreamerv3/pong_1_2_and_4gpus.svg">
```

Comment:
This is not the best practice as this image will always be pointing to master. So if we release 2.7 and tomorrow change the image, the old content will point to the new image on master. 

We should do sth like: 
![pong_1_2_and_4gpus](relative path to image)

Reply:
I think this is difficult doing this from the README (which is in a completely different location than the docs and does not get built together with the docs), but let me check.

Reply:
Figured it out. Never mind.